### DISCUSSION TITLE
One whole hour of recordings missing

### DISCUSSION THREAD

**Discussion by tlundste** [blakeblackshear/frigate#2493] [Ask A Question]
Running 0.9.0-rc4-amd64 in docker on a NUC7, running on Ubuntu 20.04.3 LTS.
It seemed to be going along just fine, but today, there is a full hour of recordings missing.  The folder isn't there... so no file is there.  Wondering what I can check to see why that hour is missing.

**Comment by tlundste** [comment]
Found in my network logs that the device (and every other wireless device I had) dropped off the network for right at a full hour.  The wireless outage started about 12 minutes before the top of the hour, and finished about 11 minutes before the top of the next hour.  There were only 10 or 11 minutes left in the hour... and my ZoneMinder software did record the last few minutes of that hour.  Should Frigate have created an entry for that hour, and recorded the last 10 minutes of that hour in that folder?

**Comment by ichinm** [comment]
I had same issue. Something went wrong, either network or camera itself then frigate stops recording.
I came up with a cron task to check recording, if it stopped and I still can ping the cam => restart frigate. :)

**Comment by lerman06470** [comment]
I see something similar. Many hours have zero data. The directories aren't even created.
Probably not related is the fact that the directory '10' contains data for the hour ending at 7:00.
Frigate (0.12.0-da3e197)

**Reply by NickM-27** [reply]
> Probably not related is the fact that the directory '10' contains data for the hour ending at 7:00.

this is expected, frigate saves recordings in UTC folder structure
